packs/data/pathfinder-bestiary-2.db/jabberwock.json

```diff
@@ -1262,20 +1262,14 @@
             },
             "dr": [
                 {
-                    "label": "Fire",
                     "type": "fire",
-                    "value": "20"
+                    "value": 20
                 }
             ],
             "dv": [
                 {
-                    "label": "Vorpal",
                     "type": "vorpal",
-                    "value": "20"
-                },
-                {
-                    "label": "Vorpal Fear",
-                    "type": "vorpal fear"
```

Comment:
We have an actual `vorpal-fear` entry in weakness types, but there's no number associated with that on the statblock so we may need to migrate that out.

Similarly `protean anatomy` should maybe become `protean-anatomy` for consistency which would need migrated.